R/geom-boxplot.r

```diff
@@ -204,7 +207,7 @@ GeomBoxplot <- ggproto("GeomBoxplot", Geom,
         size = outlier.size %||% data$size[1],
         stroke = outlier.stroke %||% data$stroke[1],
         fill = NA,
-        alpha = NA,
+        alpha = outlier.alpha %||% data$alpha[1],
```

Comment:
Thanks for catching that. I changed the default NULL and tested and it seems to work fine that way.